docs/release_notes.md

```diff
@@ -1,4 +1,11 @@
 # Subscriptions API Release Notes
+<div class='post-date'>04 November 2020 - Invoice fields</div> 
+Merchants can add [invoice](invoice) for susbscription payments without user's information.
+Fields not mandatory: consumer_name, consumer_phone_number, consumer_address_lines.
+If the fields are not filled, they are not displayed in PDF.
+Can be negative: TotalVatAmount, TotalPriceIncludingVat, PricePerUnit, PriceDiscount.
```

Comment:
these fields have snake case in the contract, don't they? If so, don't use different casing here.